.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @whiskeysierra @lukasniemeier-zalando
+* @whiskeysierra @lukasniemeier-zalando @kasmarian @msdousti
```

Comment:
Github warns `Unknown owner on line 1: make sure @msdousti exists and has write access to the repository`
Unfortunately I can't check settings.
@kasmarian Could you check?

Reply:
This should be resolved now